documentation/modules/operators/con-topic-operator-store.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='ref-topic-operator-store-{context}']
+= Topic Operator topic store
+
+[role="_abstract"]
+The Topic Operator uses persistent storage to store topic metadata describing topic configuration as key-value pairs.
+The _topic store_ is based on the Kafka Streams key-value mechanism, which uses persistent storage with local and remote components.
```

Comment:
The term `persistent storage` is misleading here since the topic operator does not use persistent storage. Maybe this should say something like this? Also, no clue what _persistent storage with local and remote components_ means.

```suggestion
The Topic Operator uses Kafka to store topic metadata describing topic configuration as key-value pairs.
The _topic store_ is based on the Kafka Streams key-value mechanism, which uses Kafka topics to persist the state.
```

Reply:
Updated as suggested. I was trying to convey the two aspects: in-memory and backup